modules/standard/JSON.chpl

```diff
@@ -80,13 +111,65 @@ module JSON {
       }
     }
 
+    /*
+      Start serializing a class by writing the character ``{``.
+
+      :arg writer: The ``fileWriter`` to be used when serializing.
+      :arg name: The name of the class type.
+      :arg size: The number of fields in the class.
+
+      :returns: A new :type:`AggregateSerializer`
+    */
+    proc startClass(writer: jsonWriter, name: string, size: int) throws {
+      writer.writeLiteral("{");
+      return new AggregateSerializer(writer, _ending="}");
+    }
+
+    /*
+      Start serializing a record by writing the character ``{``.
+
+      :arg writer: The ``fileWriter`` to be used when serializing.
+      :arg name: The name of the class type.
+      :arg size: The number of fields in the class.
+
+      :returns: A new :type:`AggregateSerializer`
+    */
+    proc startRecord(writer: jsonWriter, name: string, size: int) throws {
+      writer.writeLiteral("{");
+      return new AggregateSerializer(writer, _ending="}");
+    }
+
+    /*
+      Returned by ``startClass`` or ``startRecord`` to provide the API for
+      serializing classes or records.
+
+      Classes and records are serialized as JSON objects. For example, a
+      ``class`` or ``record`` with integer fields 'x' and 'y' with values '0'
+      and '5' would be serialized as:
+
+      .. code-block:: text
+
+        {"x":0, "y":5}
+
+    */
     record AggregateSerializer {
+      @chpldoc.nodoc
       var writer;
+      @chpldoc.nodoc
       var _parent : bool = false;
+      @chpldoc.nodoc
       var _first : bool = true;
+      @chpldoc.nodoc
       const _ending : string;
+      @chpldoc.nodoc
       var _firstPtr : c_ptr(bool) = nil;
 
+      /*
+        Serialize ``field`` named ``name``.
+
+        Serializes fields in the form '"<name>":<field>'. Adds a comma before the
```

Comment:
Maybe double backtick the ``"<name>":<field>`` or use some other formatting techniques to make it pop.